CHANGES/6591.misc

```diff
@@ -0,0 +1 @@
+Lazy load Gunicorn workers on import, decreasing import time by ~53%.
```

Comment:
Let's use the past tense here to keep the resulting changelog consistent, writing style-wise.
```suggestion
Changed importing Gunicorn to happen on-demand, decreasing import time by ~53% -- :user:`Dreamsorcerer`.
```